GameDefinition/Content/CaveKind.hs

```diff
@@ -189,7 +189,7 @@ noise2 = noise
   , citemNum      = 13 `d` 5  -- an incentive to explore the final labyrinth
   , cplaceFreq    = [("noise", 1), ("mine", 99)]
   , cstairFreq    = [("gated staircase", 100)]
-  , cdesc         = ""
+  , cdesc         = "Frost stretches across the bare rock."
   }
 shallow2rogue = rogue
```

Comment:
This one is just ordinary rogue cave, but we guarantee it always appears on level 2 (see https://github.com/LambdaHack/LambdaHack/blob/v0.7.0.0/GameDefinition/Content/ModeKind.hs#L435). So you have full control over which level depth this is to use in the description. If you imagine anything special and put it in the description, we can try and add it to the cave stats later on as well (just create an issue, if it's not some trivial stat tweak).